io/disk/htx_block_devices.py.data/Readme.txt

```diff
@@ -0,0 +1,9 @@
+HTX [Hardware Test eXecutive] is a test tool suite, which is used by various
+System p validation labs to verify the System p hardware design. It is used
```

Comment:
If HTX can be run only on ppc64 based processors, please skip for other processors